doc/man1/openssl-dsa.pod.in

```diff
@@ -115,8 +115,10 @@ This option prints out the value of the public key component of the key.
 
 =item B<-pubin>
 
-By default, a private key is read from the input file. With this option a
-public key is read instead.
+By default, a private key is read from the input.
+With this option a public key is read instead.
+If the input includes no public key but a private key,
+the public part of the private key is used.
```

Comment:
```suggestion
By default, a private key is read from the input.
With this option a public key is read instead.
If the input is a private key, its public part is used.
```
Here and below...

Reply:
Changed the last sentence here and in all other 6 doc files to:
```
If the input contains no public key but a private key, its public part is used.
```
I kept the "no public key but a private key" because it is more precise.
The point is that an input file may contain multiple keys, and private keys are considered only if it does not contain any public key.